debian/rules

```diff
@@ -103,23 +60,10 @@ override_dh_shlibdeps:
 	dh_shlibdeps -Xtestdata -Xtest
 
 debian/build.stamp:
+	[ -f VERSION ] || echo "debian snapshot +$$(dpkg-parsechangelog -SVersion)" > VERSION
 	rm -f debian/build.stamp
 	mkdir -p $(GOBIN)
-	# Build native tools first, then bootstrap all other GOOS/GOARCH combinations.
-	cd src && bash ./make.bash --no-banner
-	# For the race detector to work (go test -race), we need to install the
-	# std library with the race detector enabled. This will result in
-	# having an additional “architecture” directory, e.g. linux_amd64_race.
-	#
-	# The race detector currently only works on linux/amd64. The check within
-	# the following loop works precisely the same way src/race.bash does.
-	-${FOR_GO_ARCH} \
-		export GOARCH=$$arch; \
-		export GOOS=$$os; \
-		cd src && bash ./make.bash --no-clean; \
-		[ "$$arch" = "amd64" ] && [ "$$os" = "linux" ] && $(GOBIN)/go install -race std; \
-		cd ..; \
-	${FOR_GO_ARCH_END}
+	export GOROOT_BOOTSTRAP=$$(env -i go env GOROOT) && cd src && bash ./make.bash --no-banner
 	>debian/build.stamp
 
 opt_no_act =
```

Comment:
I mean the stuff like " $(error unrecognized build host instruction set $(DEB_HOST_ARCH_CPU)!)" on https://github.com/tianon/debian-golang/blob/tianon-beta/debian/rules#L131 -- need to add arm64 & ppc64el cases.